app/services/rad_config.rb

```diff
@@ -148,6 +148,18 @@ def twilio_verify_service_sid!
       secret_config_item! :twilio_verify_service_sid
     end
 
+    def twilio_verify_enabled?
+      boolean_config_item! :twilio_verify_enabled
+    end
+
+    def twilio_verify_all_users?
+      boolean_config_item! :twilio_verify_all_users
+    end
+
+    def twilio_verify_remember_device!
+      config_item! :twilio_verify_remember_device_days
```

Comment:
we currently have a precedent for requiring all config settings in all apps so that it's explicit - I could go either way on it but for now, following pattern